README.md

```diff
@@ -129,15 +129,15 @@ And that’s it — you should now be able to copy our code samples into your `i
 
 1. Install `Node/npm`. Below is a link to find the install method that coincides with your operating system:
 
-    - Node v4.2.3+, [Installation guides](https://nodejs.org/en/download/)
+    - Node v8.14.1, [Installation guides](https://nodejs.org/en/download/)
```

Comment:
Is it worth saying that versions _between_ 4.2.3 and 8.14.1 are necessary? Or is it better as is?

Reply:
I think it's simpler and introduces fewer variables if we stick with one, the version we have in our `.nvmrc` file.